utils/contract-utils.js

```diff
@@ -14,6 +14,32 @@ const isValidContract = function (contract) {
 	return true;
 };
 
+const isValidLibrary = function (libraries) {
```

Comment:
What will happen if I happen to input something like string or number, or a boolean?